easybuild/easyconfigs/p/PythonPlus/PythonPlus-3.7.4-foss-2018b-v20.02.1.eb

```diff
@@ -24,765 +26,22 @@ dependencies = [
-    }),
-    ('biopython', '1.74', {
-        'modulename': 'Bio',
-        'source_urls': ['https://files.pythonhosted.org/packages/source/b/biopython'],
-        'checksums': ['25152c1be2c9205bf80901fc49adf2c2efff49f0dddbcf6e6b2ce31dfa6590c0'],
-    }),
-    ('ete3', '3.1.1', {
-        'source_urls': ['https://files.pythonhosted.org/packages/source/e/ete3'],
-        'checksums': ['870a3d4b496a36fbda4b13c7c6b9dfa7638384539ae93551ec7acb377fb9c385'],
-    }),
-    ('click', '7.0', {
-        'source_tmpl': 'Click-%(version)s.tar.gz',
-        'source_urls': ['https://files.pythonhosted.org/packages/source/c/click'],
-        'checksums': ['5b94b49521f6456670fdb30cd82a4eca9412788a93fa6dd6df72c94d5a8ff2d7'],
-    }),
-    ('livereload', '2.6.1', {
-        'source_urls': ['https://files.pythonhosted.org/packages/source/l/livereload'],
-        'checksums': ['89254f78d7529d7ea0a3417d224c34287ebfe266b05e67e51facaf82c27f0f66'],
-    }),
-    ('PyYAML', '5.1.2', {
-        'modulename': 'yaml',
-        'source_urls': ['https://files.pythonhosted.org/packages/source/P/PyYAML'],
-        'checksums': ['01adf0b6c6f61bd11af6e10ca52b7d4057dd0be0343eb9283c878cf3af56aee4'],
-    }),
-    ('Markdown', '3.1.1', {
-        'source_urls': ['https://files.pythonhosted.org/packages/source/M/Markdown'],
-        'checksums': ['2e50876bcdd74517e7b71f3e7a76102050edec255b3983403f1a63e7c8a41e7a'],
-    }),
-    ('mkdocs', '1.0.4', {
-        'source_urls': ['https://files.pythonhosted.org/packages/source/m/mkdocs'],
-        'checksums': ['17d34329aad75d5de604b9ed4e31df3a4d235afefdc46ce7b1964fddb2e1e939'],
-    }),
+sources = ['%(name)s-%(version)s-foss-2018b%(versionsuffix)s-requirements.txt']
+checksums = ['f4bd09948cf6ac19aafc3daa1a5189e2d65ce644404dce42ea6614993ede82f2']
 
-    ('pysam', '0.15.4', {
-        'source_urls': ['https://files.pythonhosted.org/packages/source/p/pysam'],
-        'checksums': ['a535e15cbd6e27f4ab74cabca30ca1df7eea283cb91d3b536d47fe113fee066f'],
-    }),
-    ('PyVCF', '0.6.8', {
-        'modulename': 'vcf',
-        'source_urls': ['https://files.pythonhosted.org/packages/source/P/PyVCF'],
-        'checksums': ['e9d872513d179d229ab61da47a33f42726e9613784d1cb2bac3f8e2642f6f9d9'],
-    }),
-    ('xgboost', '0.90', {
-        'source_urls': ['https://files.pythonhosted.org/packages/source/x/xgboost'],
-        'checksums': ['d69f90d61a63e8889fd39a31ad00c629bac1ca627f8406b9b6d4594c9e29ab84'],
-    }),
-]
-
-modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+pip_ignore_installed = False
+sanity_pip_check = True
+unpack_sources = False
+use_pip = True
+use_pip_for_deps = True
+use_pip_requirement = True
 
 #
-# Run a full sanity check as opposed to just trying to load the Python package.
+# Sanity checking.
 #
-full_sanity_check = True
-
 sanity_check_paths = {
     'files': [],
-    'dirs':  ['lib/python%(pyshortver)s/site-packages'],
+    'dirs': ['lib/python3.7/site-packages']
```

Comment:
Why hard coding python version in path?

Reply:
With `'lib/python%(pyshortver)s/site-packages'` the variable doesn't get substituted and the test fails with a `the path lib/python%(pyshortver)s/site-packages` doesn't exist error.